megatron/model/transformer.py

```diff
@@ -420,8 +433,8 @@ def __init__(self, config, layer_number,
             and attention_type == AttnType.self_attn \
             and self.attn_mask_type == AttnMaskType.causal
         if self.use_flash_attn:
-            if flash_attn_unpadded_func is None:
-                raise ImportError('FlashAttention is not installed, please install with '
+            if flash_attn_unpadded_func is None and flash_attn_builder is None:
+                raise ImportError('FlashAttention is not installed and not implemented, please install with '
```

Comment:
You might want to add " or implement your own fmha" (after replacing fmha with a more clear description) after the "pip install flash-attn" in the next line.